pkg/controller/controller_test.go

```diff
@@ -80,13 +80,13 @@ func TestQueueRetryManager_retries(t *testing.T) {
 		keyFunc: func(obj interface{}) (string, error) {
 			return obj.(testObj).id, nil
 		},
-		retryFunc: func(obj interface{}, err error, count int) bool {
-			if count > 4 {
+		retryFunc: func(obj interface{}, err error, r Retry) bool {
+			if r.Count > 4 {
```

Comment:
I'll just add checking for non-zero value on this field, as other tests will unnecessarily increase the times of the test.